src/main/drivers/system_stm32f4xx.c

```diff
@@ -39,13 +39,15 @@ void systemReset(void)
     NVIC_SystemReset();
 }
 
+PERSISTENT uint32_t bootloader_request = 0;
```

Comment:
(`camelCase`)

Reply:
Ahh, I got used to another style recently and totally forgot about it :-)